porter/v1/addons.proto

```diff
@@ -28,6 +28,23 @@ enum AddonType {
   ADDON_TYPE_DEEPGRAM = 9;
 }
 
+// AddonStatus specifies the status of an addon installation
+// this is currently used to track the installation status for complex addons like deepgram and other LLM addons
+enum AddonStatus {
+  // ADDON_STATUS_UNSPECIFIED is the default value
+  ADDON_STATUS_UNSPECIFIED = 0;
+  // ADDON_STATUS_QUOTA_PENDING is the status when an addon installation is waiting for a quota upgrade
+  ADDON_STATUS_QUOTA_PENDING = 1;
+  // ADDON_STATUS_QUOTA_FAILED is the status when an addon can't be installed due to a failed quota upgrade
+  ADDON_STATUS_QUOTA_FAILED = 2;
+  // ADDON_STATUS_INFRA_PROVISIONING is the status when an addon installation is waiting for infrastructure reprovisioning
+  ADDON_STATUS_INFRA_PROVISIONING = 3;
+  // ADDON_STATUS_INFRA_PROVISIONING_FAILED is the status when an addon can't be installed due to a failed infrastructure reprovisioning
+  ADDON_STATUS_INFRA_PROVISIONING_FAILED = 4;
+  // ADDON_STATUS_DEPLOYING is the status when an addon is being deployed
+  ADDON_STATUS_DEPLOYING = 5;
```

Comment:
should this be ADDON_STATUS_INSTALLED? when the frontend makes the query, it wants to know whether a model is installed, after which it can render the deployment status